### DISCUSSION TITLE
how to handle a time column

### DISCUSSION THREAD

**Discussion by scal-lider** [adonisjs/core#2538] [Help] [answered]
Hy People, I'm confused about how to map a time column from database (I'm using postgres)

My database have a table that saves a dateTime in 2 columns: one  for date, and one for time. (Not by my choise 😭)

In my migration I know that I can user `table.time()` and `table.date()`, but in my model I can only use `@column.date` and `@column.datetime` to type my properties. 

So how do handle the time column ?
How DateTime handles only time? It use the ISO first day as date?

**Comment by thetutlage** [comment, marked as answer]
The model decorators are used to convert the `date` and `datetime` values to an Instance of Luxon date time objects. I don't think, you can create a luxon datetime object just from the time. Hence you can create a regular string property on the model class
